test/mason/pkgconfig-tests/SKIPIF

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+
+if [[ $HOST == *chapcs* ]] ; then
```

Comment:
If we address the version-independence issue I mentioned below, we may be able to change this constraint to checking if the needed packages are available on the system, e.g.


```bash
# Check if pkg-config is available
command -v pkg-config 2> /dev/null || echo False && exit

# Check if packages are available
pkg-config --exists openssl  || echo False && exit

echo True
```

This may lead to some odd errors in certain testing configurations / platforms, but I think we want to catch those before users do.

Reply:
Does Mason give a reasonable error if pkg-config is not found (I didn't look at the code here)?

Reply:
Yes